lib/carma/client/configuration.rb

```diff
@@ -8,6 +8,10 @@ class Configuration < Salesforce::Configuration
       def service_name
         'CARMA'
       end
+
+      def mock_enabled?
```

Comment:
I think since there is a new method being added, we should have a simple unit test for it.